scanner/src/main/kotlin/storages/ClearlyDefinedStorage.kt

```diff
@@ -122,13 +112,30 @@ class ClearlyDefinedStorage(
                 coordinates.revision.orEmpty()
             )
 
-            val version = findScanCodeVersion(tools, coordinates)
-            if (version != null) {
-                listOfNotNull(loadScanCodeResult(coordinates, version))
-            } else {
-                logger.debug { "$coordinates was not scanned with any version of ScanCode." }
+            val toolsVersionsByName = tools.mapNotNull { it.withoutPrefix("$coordinates/") }
+                .groupBy({ it.substringBefore('/') }, { it.substringAfter('/') })
+                .mapValues { (_, versions) -> versions.sortedWith(AlphaNumericComparator) }
+
+            toolsVersionsByName.mapNotNull { (name, versions) ->
+                // See https://github.com/clearlydefined/service#tool-name-registry.
+                when (name) {
+                    "scancode" -> {
```

Comment:
In the preceding commit one usage of the constant `TOOL_SCAN_CODE` has been removed. Should line 122 make use of a constant? Maybe of `TOOL_SCAN_CODE` ?

Reply:
I've deliberately removed the constant as the string now only occurs once, and I found it more convenient to see the actual string values directly as part of then `when` clause. Also, as I plan to introduce more `when` cases, I otherwise would have needed to introduce constants for these, too.